multiwallet.go

```diff
@@ -614,12 +673,12 @@ func (mw *MultiWallet) NumWalletsNeedingSeedBackup() int32 {
 }
 
 func (mw *MultiWallet) LoadedWalletsCount() int32 {
-	return int32(len(mw.wallets))
+	return int32(len(mw.walletsReadCopy()))
```

Comment:
No need to make a copy of the map just to get the count of items. A simple lock/unlock would be more efficient here.